components/Profile/Business/BusinessAPI.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { API, Auth, graphqlOperation, Storage } from "aws-amplify";
+import { createBusiness, updateBusiness } from "../../../src/graphql/mutations";
+import { Business } from "../../../src/API";
+
+export default class BusinessAPI {
+  static async create(business: Business) {
+    const user = await Auth.currentAuthenticatedUser();
```

Comment:
You should also be able to get the email from the user object in redux. Doubt this is a very expensive async operation but could be avoided